designs/gozer/src/ghost.mjs

```diff
@@ -0,0 +1,96 @@
+export const ghost = {
+  name: 'gozer.ghost',
+  measurements: ['hpsToWaistBack', 'waistToFloor', 'head'],
+  draft: ({ measurements, Point, Path, points, paths, Snippet, snippets, sa, macro, part }) => {
+    const c = 0.55191502449351
+    const eyeSize = measurements.head * 0.0416
+    const size =
+      measurements.hpsToWaistBack + measurements.waistToFloor + measurements.head / Math.PI
+
+    points.middle = new Point(0, 0)
+    points.eyeLine = points.middle.shift(270, measurements.head / Math.PI / 2)
+    points.eyeLeft = points.eyeLine.shift(180, measurements.head * 0.13)
+    points.eyeRight = points.eyeLine.shift(0, measurements.head * 0.13)
+    points.top = new Point(0, -1 * size)
+    points.left = new Point(-1 * size, 0)
+    points.right = new Point(size, 0)
+    points.bottom = new Point(0, size)
+    points.topCp1 = points.top.shift(180, size * c)
+    points.topCp2 = points.top.shift(0, size * c)
+    points.leftCp1 = points.left.shift(270, size * c)
+    points.leftCp2 = points.left.shift(90, size * c)
+    points.bottomCp1 = points.bottom.shift(0, size * c)
+    points.bottomCp2 = points.bottom.shift(180, size * c)
+    points.rightCp1 = points.right.shift(90, size * c)
+    points.rightCp2 = points.right.shift(270, size * c)
+
+    points.eyeLefttop = points.eyeLeft.shift(90, size * eyeSize)
```

Comment:
Regardless of the remark about circles above, I suspect this might be a mistake.

You are multiplying `size * eyeSize` here, but I think just `eyeSize` is what's intended.

If nothing else, when I fire up the lab and run this pattern, I don't see anything. That's because the eyes are (a lot) bigger than the entire body, so the pattern gets so large that the lines get vanishingly small.

Reply:
You're right, and I'm an idiot.